.oelint-ruleset.json

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
Is it some default config, or we already adjsuted it?

Reply:
I based this config on https://github.com/3mdeb/meta-trenchboot/pull/28/files#diff-cceeba45d98af3b36ef0a37e8ffdda5ff5d1ba0084c4ad21dad516a28f2e20a4R1
I only changed `oelint.var.mandatoryvar.DESCRIPTION` everity to warning, because i feel like defining the summary is enough.

Reply:
> I only changed oelint.var.mandatoryvar.DESCRIPTION everity to warning, because i feel like defining the summary is enough.


Our feelings do not matter, we should simply adhere to the rules and DESCRIPTION is required variable: https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#required-variables

Reply:
If we do not want to do this right now because it is **a lot** of work let's say, let's at least create issue for re-enabling that check.

Reply:
Ok let's stick to that for now.